style.css

```diff
@@ -85,6 +94,19 @@ main h1 {
 }
 
 /* Inputs */
+.inputSellerClass,
```

Comment:
why don't use one classname?

Reply:
Because below I am applying different widths for these classes)